crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj

```diff
@@ -21,6 +21,15 @@
 		5F99610C1AE0CF4F0034F503 /* LaunchScreen.xib in Resources */ = {isa = PBXBuildFile; fileRef = 5F9961071AE0CF4F0034F503 /* LaunchScreen.xib */; };
 		5F99610D1AE0CF4F0034F503 /* LaunchScreen.xib in Resources */ = {isa = PBXBuildFile; fileRef = 5F9961071AE0CF4F0034F503 /* LaunchScreen.xib */; };
 		5FDE055D1B0DAA090037B82F /* AppTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 5FDE055C1B0DAA090037B82F /* AppTests.m */; };
+		C732A2902684656200AA7DEF /* BuildFile in Frameworks */ = {isa = PBXBuildFile; productRef = C732A28F2684656200AA7DEF /* SwiftPackageProductDependency */; };
+		C732A2952684694300AA7DEF /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = C732A2942684694300AA7DEF /* GoogleService-Info.plist */; };
```

Comment:
Delete the GoogleService-Info.plist file from the Xcode project navigator to remove these references.